docs/decisions/0002-permissions-and-groups.rst

```diff
@@ -0,0 +1,62 @@
+Permissions and groups
+######################
+
+Status
+======
+Accepted
+
+Context
+=======
+Not every user should have the same level of access to portal designer. At a minimum, the following groups should be
+supported:
+
+- <site>_Moderators: Can manage their site
+- Global moderators: Can manage all sites
+- Administrators: Can manage everything
+
+Staff users should be automatically granted some level of access when they login (they should not see an error page).
+
+Decision
+========
```

Comment:
Would it be worth it to make this a bulleted list of decisions?

Reply:
I have a slight preference for keeping the text in paragraph form, but could really go either way.